.eslintrc.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    jsx: true,
+    useJSXTextNode: true
+  },
+  extends: [
+    "react-app",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "prettier/@typescript-eslint"
+  ],
+  plugins: ["@typescript-eslint", "graphql"],
+  rules: {
+    "graphql/template-strings": [
+      "error",
+      {
+        // Import default settings for your GraphQL client. Supported values:
+        // 'apollo', 'relay', 'lokka', 'fraql', 'literal'
+        env: "apollo",
+        // Import your schema JSON here
+        schemaJson: require("./schema.json")
+
+        // OR provide absolute path to your schema JSON (but not if using `eslint --cache`!)
+        // schemaJsonFilepath: path.resolve(__dirname, './schema.json'),
+
+        // OR provide the schema in the Schema Language format
+        // schemaString: printSchema(schema),
+
+        // tagName is gql by default
```

Comment:
do we need to leave the comments in or can they go? i always read comments, so if they don't provide value then they're just extra stuff i have to read.

Reply:
I removed the schema and these comments in this follow up PR: EVG-7140